versions/g-/gmp.json

```
@@ -1,5 +1,10 @@
 {
   "versions": [
+    {
+      "git-tree": "79b2b7cf5ffa0ceb6b769efbfe90ebc6fb0db430",
+      "version-string": "6.2.1",
```

```suggestion
      "version-semver": "6.2.1",
```
This should be `version-semver` here. You can run ./vcpkg x-add-verion --overwrite-version `gmp `to update this.